ushahidi/platform-client
# Feature/intercom

This pull request makes the following changes:
- Expanded information sent to intercom
- Set intercom only to load when user logged in
- Set intercom to unload on user log out

Test these changes by:
- Create a new deployment
- Log in, you should see an intercom welcome message
- Check intercom interface, you should see your new user, and the user's information - name/role etc should be correct

Fixes ushahidi/platform# .

Ping @ushahidi/platform


---
This change is [<img src="https://reviewable.io/review_button.svg" height="35" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/ushahidi/platform-client/128)
